client/js/components/statement/publicStatementLists/DpPublicStatement.vue

```diff
@@ -380,10 +373,8 @@ export default {
     },
 
     headerContent () {
-      const user = this.showAuthor ? `${this.user} | ` : ''
-
```

Comment:
I think we can't remove this. since the component is used in several places and some need to display the users name